pynicotine/gtkgui/chatrooms.py

```diff
@@ -266,7 +271,7 @@ def join_room(self, msg):
             # Did not auto-join room, switch to tab
             core.chatrooms.show_room(msg.room)
```

Comment:
The activation behavior is inconsistent.

The first time it appears in the background (without changing focus). Thereafter when it is disabled/re-enabled for a second time then it grabs the tab focus.

I suspect it depends upon if the room is in the autojoin_rooms list or not.

However, that problem is not related to this PR.